assets/xml/objects/object_rb.xml

```diff
@@ -1,44 +1,44 @@
 ﻿<Root>
```

Comment:
One more small thing, could you add a comment between root and file describing what this object is? See `object_giant` for an example.